tests/aequilibrae/project/test_osm_downloader.py

```diff
@@ -0,0 +1,25 @@
+import importlib.util as iutil
+from unittest import TestCase
+from time import sleep
+from aequilibrae.project.network.osm_downloader import OSMDownloader
+
+spec = iutil.find_spec("PyQt5")
+pyqt = spec is not None
+
+
+class TestOSMDownloader(TestCase):
+    def test_do_work(self):
+        self.o = OSMDownloader([[0.0, 0.0, 0.1, 0.1]], ["car"])
+        self.o.doWork()
+        if self.o.json:
+            self.fail("It found links in the middle of the ocean")
+
+    def test_do_work2(self):
+        # LITTLE PLACE IN THE MIDDLE OF THE Grand Canyon north rim
+        self.o = OSMDownloader([[-112.185, 36.59, -112.179, 36.60]], ["car"])
+        self.o.doWork()
+        if len(self.o.json[0]["elements"]) > 1000:
+            self.fail("It found too many elements in the middle of the Grand Canyon")
+
+        if len(self.o.json[0]["elements"]) < 10:
+            self.fail("It found too few elements in the middle of the Grand Canyon")
```

Comment:
Are you sure you want to automatically run osm downloading in your automated test suite?

Reply:
That is the only way I could find to really test this.  I chose a little patch in the middle of the Grand Canyon National part that should be very stable.

Reply:
I was actually meaning the etiquette of hitting a non-profit's public API over and over. 

I'd consider getting it working and then marking as 'on-hold', or putting a "only runs one time in 10" in the test

Reply:
Oh. I see.  Done.

Reply:
I am making once every 20 times